test/TCLibTest.cpp

```diff
@@ -109,13 +109,13 @@ unittest(storeDataToSD) {
   contents = contents.substring(i + 1);
   i = contents.indexOf('\n');
   line = contents.substring(0, i);
-  String expected("04/27/2021 14:24:50,   0, 20.021, 20.000, 7.5000, 7.5000,    0, 100000.0,   0.0,   0.0");
+  String expected("04/27/2021 14:24:50,   0, 20.02, 20.00, 7.500, 7.500,    0, 100000.0,      0.0,      0.0");
```

Comment:
When thinking about printing significant digits and our recent issue #178, I think we should consider using test values that have unique values for all decimal places. Zero may be from padding or having a value that is zero.

Reply:
I believe I've addressed this. Could you review it and let me know?

Reply:
Nice job.